bgt-loader/pom.xml

```diff
@@ -221,29 +221,36 @@ SPDX-License-Identifier: MIT
                     </execution>
                 </executions>
             </plugin>
-            <!-- disabled globally for now, needs to be disabled for windows profile used in Windows MS SQL CI workflow -->
-            <!--        <plugin>
-                      <groupId>io.fabric8</groupId>
-                      <artifactId>docker-maven-plugin</artifactId>
-                      <configuration>
-                        <verbose>true</verbose>
-                        <skip>true</skip>
-                        <images>
-                          <image>
-                            <name>b3partners/brmo-bgt-loader:%l</name>
-                          </image>
-                        </images>
-                      </configuration>
-                      <executions>
-                        <execution>
-                          <id>docker-build</id>
-                          <phase>package</phase>
-                          <goals>
+            <plugin>
```

Comment:
Ik denk dat deze plugin niet veel toevoegt, hij ondersteunt bijvoorbeeld geen BuildKit voor multi platform builds. We kunnen ook met GitHub actions containers bouwen en taggen, niet alles hoeft wat mij betreft via Maven.

Reply:
Maar dat gebeurt dus nu niet, multiplatform is leuk, maar vrij zinloos (aangezien de extra plaforms exoten zijn als arm64, maar bijv. Windows ontbreekt... de platforms die ondersteund worden voor de BRMO zijn alleen Windows en Linux/amd64)

Dit zorgt ervoor dat als er een release gemaakt wordt (met Maven) er dus automatisch ook een image gepushed wordt van die tag.